app/helpers/vita_min_form_builder.rb

```diff
@@ -286,7 +286,7 @@ def vita_min_money_field(
     )
     text_field_options = standard_options.merge(
       class: (classes + ["text-input money-input"]).join(" "),
-      ).merge(options).merge(error_attributes(method: method)).merge(placeholder: '0.00')
+      ).merge(error_attributes(method: method)).merge(placeholder: '0.00').merge(options)
```

Comment:
What happens when with the change of order?

Reply:
Should have called this out! This allows me to override the placeholder text, which you can see in app/views/state_file/questions/nj_sales_use_tax/edit.html.erb:38

Because we're only allowing whole rounded numbers, and throwing an error if someone inputs a decimal (this is an ongoing discussion from a UX perspective, might change, but is our pattern for now) - it seemed important to change the placeholder text here so that the user isn't seeing `0.00` on a field that won't accept a decimal number.